lib/spack/spack/caches.py

```diff
@@ -40,12 +40,12 @@ def _misc_cache():
 def fetch_cache_location():
     """Filesystem cache of downloaded archives.
 
-    This prevents Spack from repeatedly fetch the same files when
+    This prevents Spack from repeatedly fetching the same files when
     building the same package different ways or multiple times.
     """
     path = spack.config.get('config:source_cache')
     if not path:
-        path = spack.paths.default_fetch_cache_path
+        path = spack.paths.user_repos_cache_path
```

Comment:
This value is `~/.spack/git_repos`, which isn't a good default for non-git packages.

How about we add a new `spack.paths` entry `spack.paths.local_user_mirror` and make it default to `$USER?CACHE_PATH/mirror`?

Reply:
Done!!